docs/release-notes.rst

```diff
@@ -6,6 +6,185 @@
  Release Notes
 ###############
 
+**************
+ Version 0.35
+**************
+
+Version 0.35.0
+==============
+
+**Release Date:** August 05, 2024
+
+**Breaking Changes**
+
+-  Master Configuration: Replace ``resource_manager.name`` with ``resource_manager.cluster_name``
```

Comment:
Add here:
Replace ``resource_manager.namespace`` with ``resource_manager.default_namespace``
The ``namespace`` field in the Kubernetes Resource Manager configuration is no longer supported and is replaced by ``default_namespace``. This field serves as the default namespace for deploying namespaced resources when the workspace associated with a workload is not bound to a specific namespace. If unset, the workloads will be sent to the release namespace, or the default Kubernetes namespace, "default", during non-helm determined deployments.

Reply:
@salonig23 to clarify, do you want to replace the following:

- Master Configuration: Replace ``resource_manager.name`` with ``resource_manager.cluster_name`` for better clarity and to support multiple resource managers.

with the following?

- Master Configuration: Replace resource_manager.namespace with resource_manager.default_namespace
The namespace field in the Kubernetes Resource Manager configuration is no longer supported and is replaced by default_namespace. This field serves as the default namespace for deploying namespaced resources when the workspace associated with a workload is not bound to a specific namespace. If unset, the workloads will be sent to the release namespace, or the default Kubernetes namespace, "default", during non-helm determined deployments.

or add it as a second "Master Configuration" item, like this?

- Master Configuration: Replace ``resource_manager.name`` with ``resource_manager.cluster_name`` for better clarity and to support multiple resource managers.

- Master Configuration: Replace ``resource_manager.namespace`` with ``resource_manager.default_namespace``.
The namespace field in the Kubernetes Resource Manager configuration is no longer supported and is replaced by ``default_namespace``. This field serves as the default namespace for deploying namespaced resources when the workspace associated with a workload is not bound to a specific namespace. If unset, the workloads will be sent to the release namespace, or the default Kubernetes namespace, "default", during non-helm determined deployments.

Reply:
The latter!

Reply:
ok thanks!
@corban-beaird this item goes just above tasks. i've added a suggested edit. below. you can incorporated the suggested edit and then resolve these comments.

Reply:
Awesome thanks Tara! 😄